nixos/lib/test-driver/Machine.pm

```diff
@@ -43,7 +43,7 @@ sub new {
             if defined $args->{bios};
         $startCommand .= $args->{qemuFlags} || "";
     } else {
-        $startCommand = Cwd::abs_path $startCommand;
+        #$startCommand = Cwd::abs_path $startCommand;
```

Comment:
It may be because of the `chdir` prior to running `$startCommand`. But I'm not sure if we're passing a relative path anywhere.